counting.js

```diff
@@ -1,20 +1,27 @@
 var chalk = require("chalk");
 var mysql = require("mysql");
 var config = require("./config.json");
-const vukkytils = require("./vukkytils.js");
 require("dotenv").config();
-let sql;
+var sql;
 
 const error = chalk.bold.red;
 const warn = chalk.yellow;
 const success = chalk.green;
 const info = chalk.blue;
-let cheader = `[${vukkytils.getString("COUNTING")}]`;
+var servers = {};
+
+function isInt(value) {
+	let x = parseFloat(value);
+	return !isNaN(value) && (x | 0) === x;
+}
 
 module.exports = {
-	start: function() {
-		if(!config.counting.enabled) return console.log(`${cheader} ${error("Counting is disabled!")}`);
-		if(!config.misc.mysql) return console.log(`${cheader} ${error("MySQL is not enabled. MySQL is required for counting.")}`);
+	start: function(client) {
+		if(client.user.username.includes("dev")) return console.log("[counting] DEVBOT DETECTED");
+		if(client.user.username.includes("Dev")) return console.log("DEVBOT DETECTED");
```

Comment:
```suggestion
		if(client.user.username.toLowerCase().includes("dev")) return console.log("[counting] DEVBOT DETECTED");
```
You could toLowerCase() the username here.
*i have not actually tested this it might not work

Reply:
As we discussed i was lazy hah